templates/zerver/help/set-notifications-for-a-single-stream.md

```diff
@@ -0,0 +1,20 @@
+# Set notifications for a single stream
+
+If you would like to have different notification settings for different streams,
+ you can set notifications for a single stream.
```

Comment:
> you can set notifications for a single stream.

This part of the sentence is kind of redundant. I think you explain this feature without this.